Cargo.toml

```diff
@@ -18,6 +18,7 @@ validator = { version = ">=0.11, <=0.12", features = ["derive"] }
 serde = "1"
 serde_urlencoded = "0.7"
 serde_json = "1"
+serde_qs = { version = "0", features = ["actix"] }
```

Comment:
Non-stable 0.x.x crates can change public API in minor release and I think should be locked in minor version